api/app/signals/apps/api/generics/permissions.py

```diff
@@ -1,10 +1,9 @@
 # SPDX-License-Identifier: MPL-2.0
 # Copyright (C) 2019 - 2021 Gemeente Amsterdam
-from django.conf import settings
 from rest_framework import exceptions
 from rest_framework.permissions import BasePermission, DjangoModelPermissions
```

Comment:
Aside: I still don't like the naming of these in DRF. This leads to naming where sometimes a thing called permission is just a Django style permission and sometimes a more complicated object that has knowledge of HTTP methods etc.

Reply:
This is a DRF naming convention therefore I think it is better to keep it inline with that naming convention.